package.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "The H5BP Team",
   "description": "Boilerplate configurations for the Apache HTTP server",
-  "version": "2.14.0",
+  "version": "2.15.0",
```

Comment:
Can you change the version in packge-lock.json too?

Reply:
Oh, good catch!

Reply:
Done 👍